extensions/sql-database-projects/src/models/project.ts

```diff
@@ -823,6 +823,55 @@ export class Project implements ISqlProject {
 		await this.addToProjFile(sqlCmdVariableEntry);
 	}
 
+	/**
+	 * Adds DatabaseSource property to the project
```

Comment:
I think doc here and below for remove should be a little bit more specific. The property may already be there, so I think we should just say "Adds additional database source name to the collection of existing sources." and for remove - it doesn't really remove a property form the project, it removes specified source name from the list and _maybe_ removes the property, if there are none left, I assume.